lib/standard/stream.nit

```diff
@@ -405,66 +420,87 @@ abstract class BufferedReader
 		return c
 	end
 
-	# Peeks up to `n` bytes in the buffer, returns an empty string on EOF
+	fun buffer_reset do
+		_buffer_length = 0
+		_buffer_pos = 0
+	end
+
+	# Peeks up to `n` bytes in the buffer
 	#
 	# The operation does not consume the buffer
 	#
 	# ~~~nitish
-	# 	var x = new FileReader("File.txt")
-	#	assert x.peek(5) == x.read(5)
+	# 	var x = new FileReader.open("File.txt")
+	# 	assert x.peek(5) == x.read(5)
```

Comment:
you do not need to indent in a fence (in fact you shouldn't because indent is kept)